whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -224,13 +225,14 @@ class ElasticSearch {
                     log.warn("Could not get external doc ${id} for ${document.getShortId()}, skipping...")
                 }
             }
-            whelk.jsonld.embellish(document.data, referencedData, true)
+            boolean filterOutNonChipTerms = true // Consider using false here, since cards-in-cards work now.
```

Comment:
Might be good to explain why cards-in-cards is a good/bad thing here.

Reply:
Agree. And should it be changed to false now?

Reply:
Not in this change, as we need to verify that it doesn't alter any behaviour unexpectedly more thoroughly.